shared/api/network_peer.go

```diff
@@ -61,6 +96,14 @@ type NetworkPeer struct {
 	// Example: network1
 	TargetNetwork string `json:"target_network" yaml:"target_network"`
 
+	// lxdmeta:generate(entities=network-peering; group=peering-properties; key=status)
+	// The only supported keys are `user.*` custom keys.
```

Comment:
Is this line copy&paste? We don't say anything like this in the current table.

Reply:
fixed